sharaf/src/ba/sake/sharaf/ResponseWritable.scala

```diff
@@ -81,9 +89,7 @@ object ResponseWritable {
 
   given ResponseWritable[doctype] with {
     override def write(value: doctype, exchange: HttpServerExchange): Unit =
-      exchange.getResponseSender.send(
-        Array(StandardCharsets.UTF_8.encode(s"<!DOCTYPE ${value.s}>"), StandardCharsets.UTF_8.encode(value.render))
-      )
+      exchange.getResponseSender.send(value.render)
```

Comment:
This breaks it since <!DOCTYPE > is not part of `render`

Reply:
Actually my code was wrong, since I wanted to do:
```scala
      exchange.getResponseSender.send(
        Array(StandardCharsets.UTF_8.encode(s"<!DOCTYPE ${value.s}>"), StandardCharsets.UTF_8.encode(value.content))
      )
```

The advantage is that it doesn't build a intermediate `String`.

Reply:
In your impl it returned doctype tag twice, see the tests.  
I wouldnt microoptimize sharaf yet, at least not without perf tests